homework/week-0/AllisonBojarski/AnimalSort(HWproblem#1)/src/nyc/c4q/alizinha/Animal.java

```diff
@@ -0,0 +1,30 @@
+package nyc.c4q.alizinha;
+/**
+ * Created by c4q-Allison on 5/1/15.
+ */
+public class Animal implements Comparable<Animal>
+{
+    String species;
+
```

Comment:
A constructor would be good here to save you some lines of code later on when you create instances of animal. Another approach would be implementing a setSpecies() mutator method to compliment your getSpecies() method.